### PR TITLE
Adds some tips for most of my changes to the game

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Adds some tips for most of my changes to the game.

## Why It's Good For The Game

Player visibility is important to see what cool features there are and tips are kind of, well, poorly mantained, honestly. Like, the last one was about bitrunning.

## Changelog

:cl:
qol: Added some tips of the round.
/:cl:

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -289,3 +289,26 @@ You don't need to destroy a Spacecoin machine to make your funds stop draining.
...
+Biological armor will protect your limb from a zombie's infective attack, unless the limb's more damaged than the armor value. Thick material, such as firesuits and eva suits, also partially protects, preventing at least the first attack from infection.
+In a pinch, you can reduce bleeding or burn infection with several commonplace reagents - flour, salt, saltwater can all be splashed onto the wound to stall for time, and tea can be drank for a boost to your body's defences.
+First-aid analyzers double the speed of wound treatment on injuries, alongside giving out normal and improvised instructions for treatment.
+Syndicate Duffelbags are a lot quicker to zip/unzip, have significantly less slowdown, and can carry up to two of various bulky, objective-related items, such as fire axes, guns, or gibtonite. Examine them closely to see all the possibilities.
+The coroner's surgical tools can be examined to notice the cruel implement label, which speeds up surgery on corpses and slows it on not-yet-corpses. A few other items also have it.
+As a Coroner, remember taht your autopsy scanner also works as an advanced health analyzer on right-click, but only for corpses.
```
Comment (Sadboysuss) [inline strings/tips.txt:310]: That

File: strings/tips.txt [hunk 2]
```diff
@@ -289,3 +289,26 @@ You don't need to destroy a Spacecoin machine to make your funds stop draining.
 As a Bitrunner, upgrading your quantum server will increase rewards and reduce downtime.
 As a Bitrunner, your avatar has a domain info ability which will give you clues to help complete virtual domains.
 Bitrunning is a crime.
+You can right click someone with wire cutters, jaws of life, and box cutters to instantly snap cablecuffs or zipties. The jaws can even instantly break handcuffs!
+You can alt-click tank transfer valves to remove a tank from them.
+You can use a multitool in your hand to track the area's local APC.
+Some items, when examined, are labeled as 'crafting components', which means you can smack them with another item to directly construct a recipe. Try using an igniter on a rod!
+You can further the cycle of life by having two adult plushies play with eachother, creating a smaller junior child plushie.
+Most species have only 32 teeth for use in dental implants. Moths have none. Lizards have seventy five.
+As a changeling, Repurposed Glands will break bolas, disable stuns, and give you a hefty speed boost at the cost of the use of your arms, which includes the ability to open restricted airlocks.
```
Comment (MrMelbert) [inline strings/tips.txt:298]:
```suggestion
As a Changeling, Repurposed Glands will break bolas, disable stuns, and give you a hefty speed boost at the cost of the use of your arms - which includes the ability to open restricted airlocks.
```

File: strings/tips.txt [hunk 3]
```diff
@@ -289,3 +289,26 @@ You don't need to destroy a Spacecoin machine to make your funds stop draining.
...
+As a Geneticist, you should usually save all mutations you unlock. Negative mutations will increase your genetic stability, allowing you to keep more positive mutations.
+You can craft peg limbs and crutches with wood for use in dire circumstances. The latter are also available in medical vendors.
+As a Heretic, you can also sacrifice cultists, rewarding you with a knowledge point and one of three unique, powerful rewards.
+As a cultist, when someone sacrifices a heretic, you will unlock one of three powerful and unique items to be created in every cultist's forge, archive, or altar.
```
Comment (MrMelbert) [inline strings/tips.txt:302]:
```suggestion
As a Cultist, if you manage to sacrifice a Heretic, you will unlock one of three powerful and unique items to be created in every one of your cult's forges, archives, or altars.
```

File: strings/tips.txt (hunk 1 above)
Comment (Jacquerel) [inline strings/tips.txt:310]:
```suggestion
As a Coroner, remember that your autopsy scanner also works as an advanced health analyzer on right-click, but only for corpses.
```

File: strings/tips.txt [hunk 4]
```diff
@@ -289,3 +289,26 @@ You don't need to destroy a Spacecoin machine to make your funds stop draining.
...
+As a Cultist, when you sacrifice a heretic, they will be bound inside a powerful haunted longsword. Cultists, or even non-cultists, can unbind the blade, unlocking its powers and abilities, but also allowing the blade to act of its own free will.
```
Comment (MrMelbert) [inline strings/tips.txt:303]:
```suggestion
As a Cultist, when you sacrifice a Heretic, they will be bound inside a powerful haunted longsword. Anyone can then unbind the blade, unlocking its powers and abilities, but also allowing the blade to act of its own free will.
```

**Review by Jacquerel — CHANGES_REQUESTED** [review]

File: strings/tips.txt [hunk 5]
```diff
@@ -289,3 +289,26 @@ You don't need to destroy a Spacecoin machine to make your funds stop draining.
...
+As a Syndicate Monkey, you're explosively allergic to species transformations and should probably avoid them.
+Biological armor will protect your limb from a zombie's infective attack, unless the limb's more damaged than the armor value. Thick material, such as firesuits and eva suits, also partially protects, preventing at least the first attack from infection.
+In a pinch, you can reduce bleeding or burn infection with several commonplace reagents - flour, salt, saltwater can all be splashed onto the wound to stall for time, and tea can be drank for a boost to your body's defences.
+First-aid analyzers double the speed of wound treatment on injuries, alongside giving out normal and improvised instructions for treatment.
+Syndicate Duffelbags are a lot quicker to zip/unzip, have significantly less slowdown, and can carry up to two of various bulky, objective-related items, such as fire axes, guns, or gibtonite. Examine them closely to see all the possibilities.
```
Comment (MrMelbert) [inline strings/tips.txt:308]:
```suggestion
Syndicate Duffelbags are a lot quicker to zip and unzip, have significantly less slowdown, and can carry up to two of various bulky, objective-related items - such as fire axes, guns, or gibtonite. Examine them closely to see all the possibilities.
```

File: strings/tips.txt [hunk 6]
```diff
@@ -289,3 +289,26 @@ You don't need to destroy a Spacecoin machine to make your funds stop draining.
...
+The coroner's surgical tools can be examined to notice the cruel implement label, which speeds up surgery on corpses and slows it on not-yet-corpses. A few other items also have it.
```
Comment (MrMelbert) [inline strings/tips.txt:309]:
```suggestion
The Coroner's surgical tools are considered 'cruel implements', which speeds up surgery on corpses but slows it on not-yet-corpses. A few other items also have it.
```

File: strings/tips.txt [hunk 7]
```diff
@@ -289,3 +289,26 @@ You don't need to destroy a Spacecoin machine to make your funds stop draining.
...
+As a Coroner, remember taht your autopsy scanner also works as an advanced health analyzer on right-click, but only for corpses.
+As the Captain, remember that your sabre deals an extra 5 damage to assistants (as long as they have their original, organic liver). It also makes them scream on hit.
```
Comment (MrMelbert) [inline strings/tips.txt:311]:
```suggestion
As the Captain, your sabre deals extra damage to Assistants (as long as they have their original liver).
```

File: strings/tips.txt [hunk 8]
```diff
@@ -289,3 +289,26 @@ You don't need to destroy a Spacecoin machine to make your funds stop draining.
...
+You can automatically extract and retract arm implants by 'activating' the empty arm they're on. This includes integrated toolsets, cursed katanas, and vorpal scythes.
```
Comment (MrMelbert) [inline strings/tips.txt:312]:
```suggestion
You can automatically extract and retract arm implants by 'activating' the empty hand they're on. This includes integrated toolsets, cursed katanas, and vorpal scythes.
```

**Comment by carlarctg** [comment]
*scream

**Comment by Loafin34** [comment]
what about modsuit energy shield modules being multi-toolable for separate colors?

File: strings/tips.txt [hunk 9]
```diff
@@ -289,3 +289,26 @@ You don't need to destroy a Spacecoin machine to make your funds stop draining.
 As a Bitrunner, upgrading your quantum server will increase rewards and reduce downtime.
 As a Bitrunner, your avatar has a domain info ability which will give you clues to help complete virtual domains.
 Bitrunning is a crime.
+You can right click someone with wire cutters, jaws of life, and box cutters to instantly snap cablecuffs or zipties. The jaws can even instantly break handcuffs!
```
Comment (timothymtorres) [inline strings/tips.txt:292]: These need to be alphabetized

**Comment by carlarctg** [comment]
> what about modsuit energy shield modules being multi-toolable for separate colors?

that change was for hardsuits

**Comment by carlarctg** [comment]
@Fikou removed it on the modsuitification of syndie hardsuits because he hates soul

File: strings/tips.txt [hunk 10]
```diff
@@ -289,3 +289,26 @@ You don't need to destroy a Spacecoin machine to make your funds stop draining.
...
+You can craft peg limbs and crutches with wood for use in dire circumstances. The latter are also available in medical vendors.
+As a Heretic, you can also sacrifice cultists, rewarding you with a knowledge point and one of three unique, powerful rewards.
+As a Cultist, if you manage to sacrifice a Heretic, you will unlock one of three powerful and unique items to be created in every one of your cult's forges, archives, or altars.
+As a Cultist, when you sacrifice a Heretic, they will be bound inside a powerful haunted longsword. Anyone can then unbind the blade, unlocking its powers and abilities, but also allowing the blade to act of its own free will.
+As a Syndicate Monkey, you're explosively allergic to species transformations and should probably avoid them.
+Biological armor will protect your limb from a zombie's infective attack, unless the limb's more damaged than the armor value. Thick material, such as firesuits and eva suits, also partially protects, preventing at least the first attack from infection.
```
Comment (norsvenska) [inline strings/tips.txt:305]:
```suggestion
Biological armor will protect your limb from a zombie's infective attack, unless the limb's more damaged than the armor value. Armor with thick material, such as firesuits and EVA suits, also partially protects, preventing at least the first attack from infection.
```

**Comment by TJatPBnJ** [comment]
Splashing saltwater on burns makes infections worse, not better.

**Pull request merged**